v201603/traffic_estimator.go

```diff
@@ -1,9 +1,149 @@
 package v201603
 
+import "encoding/xml"
+
 type TrafficEstimatorService struct {
 	Auth
 }
 
 func NewTrafficEstimatorService(auth *Auth) *TrafficEstimatorService {
 	return &TrafficEstimatorService{Auth: *auth}
 }
+
+type Keyword struct {
```

Comment:
@Machiel, I would prefer we didn't duplicate structs here. Since this is essentially the same as KeywordCriterion, I would much rather we use that. Then in the future if for whatever reason the Keyword object is updated we only have to do it in 1 place. I see why you did it and it makes sense, but you could move the namespace to the KeywordCriterion or utilize the MarshalXML method to add in the namespace, see line 363 of [adwords_user_list.go](https://github.com/colinmutter/gads/blob/master/v201603/adwords_user_list.go)